src/com/twitter/intellij/pants/service/project/resolver/PantsCreateModulesExtension.java

```diff
@@ -112,10 +108,15 @@ public void resolve(
 
     final DataNode<ModuleData> moduleDataNode = projectInfoDataNode.createChild(ProjectKeys.MODULE, moduleData);
 
-    DataNode<ProjectSdkData> sdk = ExternalSystemApiUtil.find(projectInfoDataNode, ProjectSdkData.KEY);
-    if(sdk != null){
-      ModuleSdkData moduleSdk = new ModuleSdkData(sdk.getData().getSdkName());
-      moduleDataNode.createChild(ModuleSdkData.KEY, moduleSdk);
+    if(targetInfo.isPythonTarget()) {
+      // FIXME this happens to work becuse a SDK with the matching name is created later. See PantsPythonSetupDataService.java:77
+      moduleDataNode.createChild(ModuleSdkData.KEY, new ModuleSdkData("python")); //FIXME
```

Comment:
Is it a good name for sdk if we have separate sdk for project or even some modules? Can we generate sdk with custom name and just set it?